scenes/enumerated/problem_12662/english_text.py

```diff
@@ -0,0 +1,9 @@
+taskNumberString = "P. 12662"
+mianish_str = "single digit"
+erknish_str = "double digits"
+table_col_label1_str = "number of digits\\\\in one count"
```

Comment:
Can you explain the meaning of `in one count`?

Reply:
It was mistake. 
`number of digits\\\\in one number` or `count of digits\\\\in one number`
Which one is right?

Reply:
The first one seems more correct, but I don't like the usage of the word `number` twice with different meanings. Can you try to make it better? If no, the first one seems fine.